scripts/dts/python-devicetree/src/devicetree/edtlib.py

```diff
@@ -1321,6 +1321,97 @@ def gpio_hogs(self) -> list[ControllerAndData]:
 
         return res
 
+    @property
+    def maps(self) -> list[ControllerAndData]:
```

Comment:
This is adding a new user-facing API to edtlib without documenting it. Please compare this with the way other properties (like `gpio_hogs`) are written and documented and add the missing documentation.

It should look like this:

```py
    @property
    def maps(self) -> list[ControllerAndData]:
        "See the class docstring"
        ...
```

And then the class docstring for `Node` should be extended to document the `maps` property appropriately.

It's hard for me to check if the function is working correctly if its behavior is not documented. It's also necessary to document all the public APIs so users of this python package know how they are supposed to work.

Reply:
Added description. Please check it.